mosip-context.json

```diff
@@ -40,6 +40,18 @@
 
         "@context": {"value": "rdf:value", "lang": "@language"}
     },
+    "firstName": {
```

Comment:
it should be based on identity json and check the policy also it should be in sync like if its define as fullName then it should be identical in all the places.

Reply:
updated changes